gamma/2.1/Dockerfile

```diff
@@ -0,0 +1,107 @@
+# base image
+FROM ubuntu:bionic as app
+
+# for easy upgrade later. ARG variables only persist during image build time
+ARG GAMMA_VER="2.1"
+
+# metadata
+LABEL base.image="ubuntu:bionic"
+LABEL dockerfile.version="2"
+LABEL software="centroid"
+LABEL software.version="v2.1"
+LABEL description="Gene Allele Mutation Microbial Assessment"
+LABEL website="https://github.com/rastanton/GAMMA/"
+LABEL license="Apache 2.0"
+LABEL maintainer="Jill Hagey"
+LABEL maintainer.email="jvhagey@gmail.com"
+
+# install dependencies
+RUN apt-get update && apt-get -y --no-install-recommends install \
+    pigz \
+    rsync \
+    unzip \
+    tar \
+    python3.7 \
+    python3-pip \
+    gzip \
+    wget \
+    git \
+    curl \
+    make \
+    g++ \
+    ncbi-blast+ \
+    hmmer && \
+    apt-get autoclean && \
+    rm -rf /var/lib/apt/lists/*
+
+#install biopython
+RUN pip3 install biopython cgecore tabulate
+
+# hmmer 3.3 since amrfinder requires 3.2 or higher (apt version is 3.1)
+# apt dependencies: make, g++
+RUN wget http://eddylab.org/software/hmmer/hmmer-3.3.tar.gz && \

```

Comment:
Can you make the hmmer version an `ARG`? (Yes, I know that's not how the 1.4 version did it)

Reply:
alternatively, you can install `hmmer` 3.3 via `apt` if you upgrade the base image to `ubuntu:focal`. Might simplify the install, but this works too